pack/fhv/fhvc.json

```diff
@@ -570,4 +570,58 @@
     "traits": "Item. Tome.",
     "type_code": "asset"
   }
-]
\ No newline at end of file
+  {
+    "code": "10738",
+    "encounter_code": "myconids",
+    "encounter_position": 1,
+    "enemy_damage": 0,
+    "enemy_evade": 3,
+    "enemy_fight": 2,
+    "enemy_horror": 1,
+    "faction_code": "mythos",
+    "flavor": "\"...and their spores smell of rotting meat.\"\n-Dr. Rosa Marquez, Miskatonic Survey, 1926",
```

Comment:
```suggestion
    "flavor": "\"...and their spores smell of rotting meat.\"\n - Dr. Rosa Marquez, Miskatonic Survey, 1926",
```

Reply:
slash+n+dash = bullet point for treacheries that give you those diamond bullets, so spaces just to avoid that.